lecli/cli.py

```diff
@@ -14,6 +15,56 @@ def cli():
 
 
 @cli.command()
+def getteams():
+    """Get teams that is associated with this account"""
+    team_api.get_teams()
+
+
+@cli.command()
+@click.argument('teamid', type=click.STRING, default=None)
+def getteam(teamid):
+    """Get team with the provided id"""
+
+    if id is not None:
+        team_api.get_team(teamid)
+    else:
+        click.echo("Example usage: lecli getteam 1234-5123-4123")
```

Comment:
Yeah we can remove it, this is automatically done by click library as I have discovered a bit later than this.